packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -83,6 +84,7 @@ export const EventMeta = ({
     eventMetaTimezoneSelect?: string;
   };
   locale?: string | null;
+  timeZones?: string[];
```

Comment:
Below we have type `Timezone[];` but here `string[]` - we need `Timezone[]` everywhere.